### DISCUSSION TITLE
Way to include script parts in documentation

### DISCUSSION THREAD

**Discussion by hograthm** [Pawamoy/shellman#43] [Q&A] [answered]
I might missing something

Let's imagine a super-simple bash script:
```bash
#!/bin/bash

## \brief My super simple script

## This instruction is needed for CI/CD
set -xeo pipefail
```

This will render only `\brief` and `This instruction is needed for CI/CD`

Is there a way to include actual code as part of my documentation?
I know, I can use `\example` in this case, but I really want to avoid duplicating code in comments for simpler refactoring.

**Comment by pawamoy** [comment, marked as answer]
Currently, no, there's no way to do that. The tool only picks up documentation comments, not code. But feel free to open a feature request :smile: Make sure to provide a detailed use-case :slightly_smiling_face:
